### ISSUE TITLE
IGC and KML

### ISSUE THREAD

**Issue by ZIRminator** [fhorinek/SkyDrop#166]
is it possible to save both igc and kml to the drop?

i also have a little problem with my logs. the altitude is always a view meters to high:

https://dl.dropboxusercontent.com/u/29303320/00-1316.KML

you can see it on start/landing. i thik i am doing something wrong with the config?

**Comment by fhorinek** [comment]
it is not possible right now.
KML is using altitude1, make sure that is set with correct QNH

**Comment by ZIRminator** [comment]
ok thank you fhorinek, Im gonna figure it out tomorrow!

**Comment by ZIRminator** [comment]
FW2320 worked very well tody!

but one thing... in the following jpg you can see that the flight path isnt smooth. is it because the gps request interval (hardware) or is it a software "problem"? didnt see such "stairs" on friends' logs with other varios. btw it was on every FW with log-function

![skydrop](https://cloud.githubusercontent.com/assets/14094559/14688319/51836468-0742-11e6-9f08-5754de025dcb.jpg)

**Comment by ZIRminator** [comment]
nobody with this interval problem?

**Comment by fhorinek** [comment]
@ZIRminator try it now with the latest release

**Comment by ZIRminator** [comment]
same problem as you can see on the first pic.
on the second picture you can see my valley crossing and on the third a friends crossing log with a skytraxx

![skydrop_gps_request_interval](https://cloud.githubusercontent.com/assets/14094559/15456776/0ca9ce70-207b-11e6-9760-082990a329ec.jpg)
![skydrop_gps_request_interval_02](https://cloud.githubusercontent.com/assets/14094559/15456777/0cac35fc-207b-11e6-9df1-c59addacd97f.jpg)
![skydrop_gps_request_interval_03](https://cloud.githubusercontent.com/assets/14094559/15456778/0cae8bb8-207b-11e6-86b4-18641382ae7a.jpg)

**Comment by fhorinek** [comment]
please attach the source file

**Comment by ZIRminator** [comment]
[02-1401.zip](https://github.com/fhorinek/SkyDrop/files/277183/02-1401.zip)

**Comment by fhorinek** [comment]
what tool are you using to convert igc to kml?

**Comment by ZIRminator** [comment]
- IGC Flight Replay
- http://www.dhv-xc.de/
- www.paraglidingearth.com/igctokml/

**Comment by fhorinek** [comment]
They are using GPS or baro altitude?

**Comment by ZIRminator** [comment]
to be honest i dont know. but no matter which flight you take from dhv-xc, they are all smooth. is that an important info then?

**Comment by fhorinek** [comment]
GPS altitude is updated once in 4 seconds right now. I will change it.

**Comment by ZIRminator** [comment]
how far can you go down with the gps update ratio? does it have effects on battery life then?

**Comment by fhorinek** [comment]
not really, I think I had some issues with the parsing speed, but I will try to increase it.

**Comment by ZIRminator** [comment]
still same problem :( did you try to "increase the parsing speed"?

![skydrop](https://cloud.githubusercontent.com/assets/14094559/17560491/8deeeee4-5f22-11e6-8dbe-9717d7e37f93.jpg)

IGC:
https://dl.dropboxusercontent.com/u/29303320/zirminator.IGC

**Comment by fhorinek** [comment]
Yes I did, please wait for the next release.
